src/goCover.ts

```diff
@@ -305,3 +309,19 @@ export function toggleCoverageCurrentPackage() {
 		});
 	});
 }
+
+export function isPartOfComment(e: vscode.TextDocumentChangeEvent): boolean {
+	let result = false
+	e.contentChanges.every(function(change, index){
+		let line = e.document.lineAt(change.range.start)
```

Comment:
What about the case when the range spans multiple lines and only the first line was part of a comment? In that case, `isPartOfComment` would return `true` instead of `false`

Reply:
For block comments of the form /* .... *,. I was thinking of the following solution

- Scan backwards from current change until start of the document to find /*. This would be necessary in cases where the block comment starts on a line after the code and changes are made to the code. In this case we would have to clear the coverage.  Example

`return 0, err /*
			one
			two
			three.
		*/`


- Scan forwards until */ is found or end of the document. 
- Check if change is within the above range.

Seems very inefficient to me.  Is there a better way of doing it? Perhaps a vscode API to identify block comments?

Reply:
Here I wasnt referring to block comments. Take the below case
```
// I am a comment
var a = 1
var b = 2
```
Now if I start selecting from the word `comment` and continue till the next line and delete the selected text, then the `line` variable in your code will have `// I am a ` and `isPartOfComment` will return `true` even though some code was deleted.

Reply:
I have updated the PR to resolve this issue.